content/parts/en/datasets.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: "Datasets"
+html_description: "Download datasets"
+introduction: "Introductory disclaimer text presenting the varying quality depending on maintainers of the collections"
```

Comment:
Shouldn't this be in the `locales` folder? 🤔

Reply:
Done, explanations are [here](https://github.com/OpenTermsArchive/opentermsarchive.org/pull/248/files#r1317467229).